.babelrc

```diff
@@ -1,7 +0,0 @@
-{
```

Comment:
I think I'd rather keep the babel config in the global config file, as it is useful for discovery by jest etc.

Reply:
After the upgrade, for some reason configuration saved in .babelrc stopped working for jest if I can remember it right - it's been a while so not sure but there was some kind of problem with discovery. This settings been moved to babel.config.js which is a global config file and is recommended by Babel after upgrade to v7 (https://babeljs.io/docs/en/configuration#whats-your-use-case, https://babeljs.io/docs/en/config-files).

Reply:
The file is here https://github.com/learningequality/studio/pull/1866/files#diff-fc10a6682269c1fefc2abe05d1500b8a

Reply:
Ah, gotcha - in that case, I'd recommend deleting the duplicate configuration from the webpack configuration (or if it needs it, explicitly importing it from the now JS compatible config file).

Reply:
Ah okay that makes sense, I will try if I can remove it - I wasn't thinking about that duplicate much and just updated that too. I am not sure if it's needed. If yes, I will import it as you suggest.

Reply:
Everything seems to work fine after removing it from the webpack config.